docs/nodes/blockspace-race.md

```diff
@@ -0,0 +1,164 @@
+* [Testnet Phases](#testnet-phases)
+* [Useful links](#useful-links)
+* [Terms and Conditions](#terms-and-conditions)
+
+## Overview
+
+Celestia is the first modular blockchain network. Its mission is
+to make deploying a blockchain as easy as deploying a new smart
+contract. Celestia introduces what is called the data availability
+layer for enabling efficient scaling and allowing L2 rollups to
+do data sampling for transactions they need.
+
+The Blockspace Race will help put the Celestia network through its
+paces to harden features and prepare node operators for mainnet launch.
+Learn more about the design of the program and its importance for the
+Celestia community at our announcement post [here](https://blog.celestia.org/the-blockspace-race/).
+
+Since Celestia is made up of two networks (consensus and data
+availability), the incentivized testnet consists of multiple node types.
+
+### Consensus Network
+
+* Validator Node: Orders transactions in the block and participate
+  in consensus.
+* Consensus Full Nodes: Syncs the chain history but does not sign
+  the transaction block.
+
+### Data Availability Network
+
+* Bridge Node: Bridges blocks between the data availability network
+  and the consensus network.
+* Light Node: Conducts data availability sampling on the data
+  availability network.
+* Full Storage Node: Stores all the data but does not connect to
+  the consensus network.
+
+Participants have been selected to run a node type from the above
+list, with a limit of one node per participant. The Blockspace Race
+will consist of five phases. Each phase will have different tasks
+depending on which node type participant will operate. Points are earned
+by completing these tasks, with additional bonus tasks that the team
+will evaluate at their sole discretion.
+
+The Celestia Labs team will provide all information, support,
+announcements and updates on [Discord](https://discord.com/invite/YsnTPcSfWQ)
+and [official website](https://celestia.org). We also encourage
+participants to familiarise themselves with the [Supplemental Incentivized
+Testnet Terms](https://docs.celestia.org/community/itn-tos/), which
+govern participation in the program.
+
+> **NOTE**: Node operator tasks will include three data availability node subtypes:
+  Bridge Nodes, Full Storage Nodes and Light Nodes. If you have been selected
+  to run one of the three data availability node subtypes, please make sure
+  to only perform the tasks corresponding to your node type. You will only
+  receive points for completed tasks corresponding to your node types.
+  If there is no indication that a node operator task is specific to any
+  of the three subtypes (such as the Bonus tasks), you are eligible for
+  that task also.
+
+### Getting Started
+
+* All participants should join [Celestia Discord](https://discord.gg/celestiacommunity).
+* All participants must create a profile on [Knack portal](https://www.knack.com/).
```

Comment:
All participants will receive access to their profile on [Knack portal](https://celestia.knack.com/theblockspacerace#testnet-portal/).